Pendulum/Views/SettingsList.swift

```diff
@@ -65,9 +68,11 @@ struct SettingsList: View {
                 Section(header: Text("Icon Badges")) {
                     Toggle("Show for unwritten responses", isOn: $badgeRemindersToWriteLetters.animation())
                     Toggle("Show for unposted letters", isOn: $badgeRemindersToPostLetters.animation())
+                        .disabled(ignoreSendStatus)
                 }
                 
                 Section(footer: Text("With Quick Entry, you won't be prompted for notes when logging a written or sent letter. You can add those later by tapping on the entry.")) {
+                    Toggle("Skip posting letters", isOn: $ignoreSendStatus)
```

Comment:
Debating 'Skip Posting Letters. 

equally bad, but it might trigger some thoughts:
- 'Track posting letters'  enabled by default,